x-pack/plugins/task_manager/server/lib/calculate_health_status.ts

```diff
@@ -19,6 +19,11 @@ export function calculateHealthStatus(
 ): { status: HealthStatus; reason?: string } {
   const now = Date.now();
 
+  // if stats are empty, return a warning
+  if (isEmpty(summarizedStats.stats)) {
```

Comment:
I still see `no health stats available` message at startup (it's now a warning).

I understand that the _health URL_ must report something when called, but for the status observables there's no need IMO (on our `status_service` we already consider plugins _unavailable_ until they report otherwise`).

@ymao1 I created a draft PR that addresses this by filtering out "empty" status emissions. It'd be nice if you could take a look to make sure I'm not causing undesired side effects:

https://github.com/elastic/kibana/pull/171012

Reply:
Thanks @gsoldevila! That makes a lot of sense to me. I left one comment but the approach LGTM